tools/IoTEdgeDevOps/DevOpsLib/BugWiqlQueries.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace DevOpsLib
+{
+    public static class BugWiqlQueries
+    {
+        public const string PrioritizedBugTemplate =
+@"SELECT
+    [System.Id],
+    [System.WorkItemType],
+    [System.Title],
+    [System.AssignedTo],
+    [System.State],
+    [System.Tags]
+FROM workitems
+WHERE
+    [System.TeamProject] = @project
+    AND [System.WorkItemType] = 'Bug'
+    AND [System.AreaPath] = 'One\IoT\Platform\{AREA}'
+    AND NOT [System.State] CONTAINS 'Done'
+    AND NOT [System.State] CONTAINS 'Removed'
+    AND [Microsoft.VSTS.Common.Priority] = {PRIORITY}";
+
+        public const string PrioritizedStartedBugTemplate =
```

Comment:
define PrioritizedStartedBugTemplate by using PrioritizedBugTemplate

Reply:
I intentionally didn't do this because I thought continuing this practice as we change / add wiql queries would be error prone and less readable. I think it is fine either way though so I'll change.

Reply:
On second thought, I'm scared that implementing it this way will make it harder to work with. I think it best to not make this change. Do you still disagree?

Reply:
not a big deal to me.